README.md

```diff
@@ -131,6 +132,29 @@ archive { '/tmp/test100k.db':
 }
 ```
 
+### Puppet URL
+
+Currently there isn't support for using ```puppet:\\``` style url in the source parameter.  It is still possible, but it would be done in two steps:
```

Comment:
Hate to sound like a pedant, but the first part of the sentence should be "Currently there is no support for using a .." - important for ESL readers.